### PR TITLE
[Improvement-12312]add the unixtime processing

### PR DESCRIPTION
**Pull request by seagle-yuan**
[Improvement][api-server] add unixtime parameter #12312 

## Purpose of the pull request

Related : #12312 

## Verify this pull request

This change added tests and can be verified as follows:

Added 
dolphinscheduler-dao tests  org.apache.dolphinscheduler.plugin.task.api.parser.TimePlaceholderUtils.timePlaceHolderForCalculateTime
for calculateTime

### CODE REVIEW

File: dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/parser/TimePlaceholderUtils.java [hunk 1]
```diff
@@ -18,35 +18,7 @@
...
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.SUBTRACT_CHAR;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.SUBTRACT_STRING;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.THIS_DAY;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.TIMESTAMP;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.WEEK_BEGIN;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.WEEK_END;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.WEEK_FIRST_DAY;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.WEEK_LAST_DAY;
-import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.YEAR_WEEK;
+import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.*;
```
Comment (SbloodyS) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/parser/TimePlaceholderUtils.java]: Please avoid import *

**Review by SbloodyS — CHANGES_REQUESTED** [review]

**Comment by SbloodyS** [comment]
Please associate the issue. @seagle-yuan

**Comment by caishunfeng** [comment]
Hi @seagle-yuan  It seems miss the UI part, please check it.

**Comment by caishunfeng** [comment]
Hi @seagle-yuan please check the fail CI.
See the code style https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/contribute/development-environment-setup.html

**Pull request closed without merging**